packages/@aws-cdk-testing/framework-integ/test/aws-codebuild/test/integ.github-webhook-workflow-job-queued.ts

```diff
@@ -0,0 +1,30 @@
+import * as cdk from 'aws-cdk-lib';
+import * as codebuild from 'aws-cdk-lib/aws-codebuild';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+
+class TestStack extends cdk.Stack {
+  constructor(scope: cdk.App, id: string) {
+    super(scope, id);
+
+    const source = codebuild.Source.gitHub({
+      owner: 'aws',
```

Comment:
Would it actually make sense to allow overriding this value to personal fork using an env var (which?) so that non cdk maintainers can run this test?